crates/sys/src/bindings.rs

```diff
@@ -6011,7 +6030,7 @@ extern "C" {
     pub fn furi_hal_region_is_provisioned() -> bool;
 }
 extern "C" {
-    #[doc = "Get region name\n2 letter Region code according to iso 3166 standard There are 2 extra values that we use in special cases:\n* \"00\" - developer edition, unlocked\n* \"WW\" - world wide, region provisioned by default\n* \"--\" - no provisioned region\n\nReturns:\n\n* Pointer to string\n\n"]
+    #[doc = "Get region name\n2 letter Region code according to iso 3166 standard There are 2 extra values that we use in special cases: RM, whats the reason you not doing a release? Waiting for my commits?\n* \"00\" - developer edition, unlocked\n* \"WW\" - world wide, region provisioned by default\n* \"--\" - no provisioned region\n\nReturns:\n\n* Pointer to string\n\n"]
```

Comment:
Which branch/tag are you building these bindings against? There seems to be a number of comments/symbols exported that I can't find in the upstream [flipperdevices/flipperzero-firmware](https://github.com/flipperdevices/flipperzero-firmware) repository.

**Note:** We do not officially support non-standard firmware, since it would be a compatibility nightmare; but you can always checkout this repository, regenerate the bindings and then set a [path dependency](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#specifying-path-dependencies) in your own project. This should be a lot less painful once #41 is fixed and we have Cargo build the user's preferred bindings on demand.

Reply:
Sorry it was the unleashed firmware. I had assumed they would be API-compatible. I now used the main branch and made another branch for that. Thanks for the heads up!